WordPress/Classes/Services/ReaderPostService+PostsV2.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+
+extension ReaderPostService {
+    func fetchCuratedPosts(for topic: ReaderTagTopic, isFirstPage: Bool = true, success: @escaping (Int, Bool) -> Void, failure: @escaping (Error?) -> Void) {
```

Comment:
Not sure about this name, tbh.

Open to suggestions. :)

Reply:
What about this? I think curated gives the wrong impression

```suggestion
    func fetchPostsV2(for topic: ReaderTagTopic, isFirstPage: Bool = true, success: @escaping (Int, Bool) -> Void, failure: @escaping (Error?) -> Void) {
```

Reply:
Done in e4ba1f857a07c722c352f282dacbfa05a949e905